client/me/purchases/manage-purchase/auto-renew-toggle/index.jsx

```diff
@@ -139,16 +142,27 @@ class AutoRenewToggle extends Component {
 		return this.props.isEnabled;
 	}
 
+	renderTextStatus() {
+		const { translate, isEnabled } = this.props;
+
+		return isEnabled ? translate( 'Auto-renew (on)' ) : translate( 'Auto-renew (off)' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Auto-renew is OFF' )` *ES Score: 10*